pkg/repository/udmrepo/kopialib/lib_repo.go

```diff
@@ -384,7 +386,11 @@ func (kor *kopiaObjectReader) Read(p []byte) (int, error) {
 
 	n, err := kor.rawReader.Read(p)
 	if err != nil {
-		return 0, errors.Wrap(err, "error to read object")
+		if errors.Is(err, io.EOF) {
```

Comment:
Why do we check whether it's io.EOF or not? Could you add some comment?

Reply:
This is the implementation of io.Reader, it returns some system wide standard errors, the caller may catch these standard errors and not treat it as an error case, for example, io.EOF.
 I will remove the error wrap for all the method of io.Reader, io.Writer, io.Closer and io.Seeker.

Reply:
Done